core/src/main/resources/defaults/clustertemplates/7.2.15/aws/search-analytics.json

```diff
@@ -0,0 +1,124 @@
+{
+  "name": "7.2.15 - Search Analytics for AWS",
+  "description": "",
+  "type": "DISCOVERY_DATA_AND_EXPLORATION",
+  "featureState": "PREVIEW",
+  "cloudPlatform": "AWS",
+  "distroXTemplate": {
+    "cluster": {
+      "blueprintName": "7.2.15 - Search Analytics"
+    },
+    "instanceGroups": [
+      {
+        "name": "master",
+        "nodeCount": 2,
+        "recoveryMode": "MANUAL",
+        "template": {
+          "attachedVolumes": [
+            {
+              "count": 1,
+              "size": 100,
+              "type": "standard"
+            }
+          ],
+          "aws": {
+            "encryption": {
+              "type": "NONE"
+            }
+          },
+          "instanceType": "m5.2xlarge",
+          "cloudPlatform": "AWS"
+        },
+        "type": "CORE"
+      },
+      {
+        "name": "gateway",
```

Comment:
Only if you run your CB locally and try deploying a DH with the new cluster definition. :smile: There is certainly no obvious validation for the consistency / compatibility of the blueprint and its associated cluster definition. It would surely break in runtime, either in CB or in CM.